src/Console/TestSuite/Constraint/ExitCode.php

```diff
@@ -27,7 +27,7 @@ class ExitCode extends Constraint
     /**
      * @var int|null
      */
-    private ?int $exitCode = null;
+    private ?int $exitCode;
```

Comment:
White it does not matter for this particular class, having a default value for the property is a good practice (even when it's initialized from the constructor). For e.g. if the class is mocked and the constructor is not executed then you would get "property not initialized" error. Same error can occur during serialization of an instance.

Reply:
Thats interesting to know. So we want to keep them then?

Reply:
I suggest we should, it doesn't cause any harm.